src/euphorie/content/widgets/templates/logo.pt

```diff
@@ -0,0 +1,63 @@
+<fieldset class="concise">
+  <p class="legend"
+     i18n:translate="label_logo"
+  >Logo</p>
+  <dfn class="infoPanel"
+       title="Information"
+       i18n:attributes="title"
+       i18n:translate="help_sector_logo"
+  >The logo will appear on the client side app that your user group will see. Make sure your image is of format png, jpg or gif and does not contain any special characters. The new logo will only become visible after you've saved your changes and published the OiRA tool.</dfn>
```

Comment:
I'd also recommend to use `webp` as it produces very good results with a lower filesize and also offers lossless compression.